### PR TITLE
Uxd 500

### PR DESCRIPTION
**Pull request by ThyeeZz**

### CODE REVIEW

File: src/components/layout/v2Layout/index.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import Footer from '../../footer/v2';
+import Header from '../../header/v2';
+
+const V2Layout = ({ header, footer, children, locale, versions }) => {
+
+  useEffect(() => {
+    if (typeof document !== 'undefined') {
+      document.getElementsByTagName('body')[0].style.backgroundColor = "#fff";
```
Comment (Tumao727) [inline src/components/layout/v2Layout/index.jsx:9]: set background color in css

File: src/components/V2Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}) => {
+  return (
+    <div className="button-wrapper">
+      {
+        type === 'link' ? (
+          <a
+            target={isExternal ? '_blank' : '_self'}
+            rel="noopener noreferrer"
+            href={href}
+            className={`${variant === 'text' ? 'text' : variant === 'outline' ? 'outline' : 'contained'
```
Comment (nameczz) [inline src/components/V2Button/index.jsx:24]: Is just variant working?

File: src/components/header/components/MobilePopupV2/index.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import './index.scss';
+
+const MobilePopUp = props => {
+  const { className = "", open, hideMask } = props;
+
+  const container = document.querySelector('.pop-up-wrapper');
```
Comment (nameczz) [inline src/components/header/components/MobilePopupV2/index.jsx:7]: document is undefined here

File: src/components/header/v2/index.jsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import MobilePopup from '../components/MobilePopUpV2';
+
+import './index.scss';
+
+
+const V2Header = ({ header, locale, versions }) => {
+  const { navlist } = header;
+  const screenWidth = useMobileScreen();
+  versions.sort((a, b) => sortVersions(a, b));
+  if (!versions.includes('v.2.0.0')) {
```
Comment (nameczz) [inline src/components/header/v2/index.jsx:19]: v2.0.0?

**Pull request merged**